src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_BOUND = 10;
+    private int iterator = 0;
```

Comment:
```suggestion
    private int iterator;
```
0 is the default value of int